README.md

```diff
@@ -382,6 +382,28 @@ class MyConfiguration {
 }
 ```
 
+### Disable worker
+
+If you need to disable worker you can do it using properties:
+```yaml
+zeebe:
+  client:
+    worker:
+      workers:
+        someWorker:
+          enabled: false
```

Comment:
First: Other parts of the doc use properties - not YAML, so I would like to keep it consistent. Like this - right?

zeebe.client.worker.workers[0].someWorker.enabled=false

Then this uses the 'type' as identifier - right? So sticking to the exymplw code below it would be

zeebe.client.worker.workers[0].foo.enabled=false
or
zeebe.client.worker.workers['foo'].enabled=false
?

Reply:
Also I wonder if 
`zeebe.client.worker.workers`
is the most intuitive way of naming this - but out of good ideas at the moment

Reply:
Based on https://github.com/camunda-community-hub/spring-zeebe/pull/221#discussion_r965570205 I think that the  best name for it should be something like this: 
```properties
zeebe.client.worker.override.some-worker-type.enabled=false
```
WDYT?